README.md

```diff
@@ -1,13 +1,12 @@
 # Privacy Cookie Bundle
 
-This bundle adds privacy cookie banner into Symfony 2 application (eZ Publish / eZ Platform).
+This bundle adds privacy cookie banner into Symfony 2 application (eZ Publish / Platform).
```

Comment:
Given that we mention eZ products at the bottom, and that the bundle is now compatible with any sf2 app, I wouldn't mention our products here.